keps/sig-instrumentation/1748-pod-resource-metrics/kep.yaml

```diff
@@ -6,7 +6,7 @@ owning-sig: sig-instrumentation
 participating-sigs:
   - sig-node
   - sig-scheduling
-status: implementable
+status: implemented
```

Comment:
IIRC, they reserve `implemented` to mean GA

Reply:
Yeah this should stay in `implementable`, we need to add a stage here:

https://github.com/kubernetes/enhancements/blob/6c2e020d4475d4170ffdeafb266a84bb444ac9a6/keps/NNNN-kep-template/kep.yaml#L26-L27

Reply:
FYI -- I'm working on cleaning up the validation logic in https://github.com/kubernetes/enhancements/pull/2409.

ref: https://kubernetes.slack.com/archives/C1L57L91V/p1612288049074600, https://kubernetes.slack.com/archives/C1L57L91V/p1612385943085000

Reply:
I'll change it